Dockerfile

```diff
@@ -71,7 +71,7 @@ COPY docker/start.sh /start.sh
 
 WORKDIR /etc/temporal
 
-ENV SERVICES="history,matching,frontend,worker"
+ENV SERVICES="--services=history --services=matching --services=frontend --services=worker"
```

Comment:
This is a breaking change and will break anybody depending on the old command line format. Are we consciously making it?